internal/driver/webhtml.go

```diff
@@ -511,6 +615,11 @@ function dotviewer(nodes) {
     return null
   }
 
+  // convert a string to a regexp that matches that string.
+  function quotemeta(str) {
+    return str.replace(/([\\\.?+*\[\](){}])/g, '\\$1')
```

Comment:
Added ^, $, |.

Also double-checked with pcre syntax that I am not missing any other special characters.